dockers/docker-lldp-sv2/reconfigure.sh

```diff
@@ -2,9 +2,8 @@
 
 set -e
 
-num_of_interfaces=32
-if_step=4
-last_if_idx=$((num_of_interfaces*if_step - if_step))
+# TODO: Listen to state database when it is ready
+interfaces=`sonic-cfggen -d -v "PORT.keys() | join(' ')"`
```

Comment:
I'd suggest to use $() instead of deprecated back ticks.

Reply:
thanks i'll update this pull request

Reply:
fixed